sys/include/net/ipv6/ext/rh.h

```diff
@@ -9,13 +10,14 @@
 /**
  * @defgroup    net_ipv6_ext_rh IPv6 routing header extension
  * @ingroup     net_ipv6_ext
- * @brief       Implementation of IPv6 routing header extension.
+ * @brief       Definitions for IPv6 routing header extension.
  * @{
  *
  * @file
  * @brief   Routing extension header definitions.
  *
- * @author  Martine Lenders <mlenders@inf.fu-berlin.de>
+ * @author  Cenk Gündoğan <cnkgndgn@gmail.com>
```

Comment:
Why this authorship addition and copyright addition above?

Reply:
I did not write the original files, Cenk did (when he was a student at FU), so I changed it to reflect the actual copyright ;-).

Reply:
Ok :+1:

Reply:
(have a look at this file's history to verify)